apps/finance/test/finance.js

```diff
@@ -636,28 +643,28 @@ contract('Finance App', ([root, owner, recipient]) => {
 
             it('can transition periods externally to remove deadlock for direct deposits', async () => {
                 const sentWei = 10
-                const prevVaultBalance = (await getBalance(vault.address)).toNumber()
+                const prevVaultBalance = parseInt(await getBalance(vault.address), 10)
 
                 await finance.tryTransitionAccountingPeriod(maxTransitions)
 
                 const receipt = await finance.send(sentWei, { gas: 3e5 })
                 const transactionId = getEventArgument(receipt, 'NewTransaction', 'transactionId')
-                const [periodId, amount, paymentId, paymentExecutionNumber, token, entity, incoming, date] = await finance.getTransaction(transactionId)
+                const data = await finance.getTransaction(transactionId)
 
-                assert.equal(amount, sentWei, 'app should have received ETH and sent it to vault')
-                assert.equal((await getBalance(vault.address)).valueOf(), prevVaultBalance + sentWei, 'app should have received ETH and sent it to vault')
+                assert.equal(data.amount, sentWei, 'app should have received ETH and sent it to vault')
+                assert.equal(parseInt(await getBalance(vault.address), 10), prevVaultBalance + sentWei, 'app should have received ETH and sent it to vault')
```

Comment:
Mentioned elsewhere, but let's avoid `parseInt()` in tests when we get BNs back.